File: DART/actividades/actividad 29-07.dart
class Animal {
  String? espesie;
  String? color;
  int? peso;
}
class Carro {
  String? modelo;
  String? color;
  int? peso;
}
class Barco {
  int? tamano;
  String? color;
  int? peso;
}

void main(){
  Animal animal = new Animal();
  animal.espesie = "perro";
  animal.color = "negro";
  animal.peso = 20;

  Carro carro = new Carro();
  carro.modelo = "nissan gtr";
  carro.color = "negro";
  carro.peso = 50;

  Barco barco = new Barco();
  barco.tamano = 40;
  barco.color = "gris";
  barco.peso = 150;
}

File: clases y trabajos/RUBY/actividades/actividad 29-07.rb
class Animal 
    attr_accessor :espesie, :color, :peso
end

class Carro 
    attr_accessor :modelo, :color, :peso
end

class Barco 
    attr_accessor :tamaño, :color, :peso
end



  animal = Animal.new
  animal.espesie = "perro"
  animal.color = "negro"
  animal.peso = 20

  carro = Carro.new
  carro.modelo = "nissan gtr"
  carro.color = "negro"
  carro.peso = 50

  barco = Barco.new
  barco.tamaño = 40
  barco.color = "gris"
  barco.peso = 150